package.json

```diff
@@ -15,6 +15,7 @@
     "generate-schema-ts": "for x in v2 planning criticals; do npx ts-node --project tsconfig_tsnode.json --esm src/lib/forms/generate_ts.ts src/schemas/$x.json > src/schemas/$x.ts; done"
   },
   "devDependencies": {
+    "@ianvs/prettier-plugin-sort-imports": "^4.0.2",
```

Comment:
This is https://github.com/ianvs/prettier-plugin-sort-imports. I chose it over https://github.com/trivago/prettier-plugin-sort-imports because it's more recently maintained, builds on the other repo, and seems actively used by people